dashboard/src/features/accounts/components/AccountUpdate.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import { BaseUpdate, FormContainer, FormSection, JsonField, NotFound } from 'features/shared/components'
+import { reduxForm } from 'redux-form'
+
+class Form extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.submitWithErrors = this.submitWithErrors.bind(this)
+
+    this.state = {}
+  }
+
+  submitWithErrors(data) {
+    return this.props.submitForm(data, this.props.item.id).catch(err => {
+      throw {_error: err}
+    })
+  }
+
+  componentDidMount() {
+    this.props.fetchItem(this.props.params.id).then(resp => {
+      if (resp.items.length == 0) {
+        this.setState({notFound: true})
+      }
+    })
+  }
+
+  render() {
+    if (this.state.notFound) {
+      return <NotFound />
+    }
+    const item = this.props.item
+
+    if (!item) {
+      return <div>Loading...</div>
+    }
+
+    const {
+      fields: { tags },
+      error,
+      handleSubmit,
+      submitting
+    } = this.props
+
+    const title = <span>
+      {'Edit account tags '}
+      <code>{item.alias ? item.alias :item.id}</code>
+    </span>
+
+    const hint = <div>
+      <p>Updating tags will overwrite existing tags. Contents must be represented as a JSON object.</p>
+      <p>
+        Account tags are used to annotate transactions where relevant. Changing the tags will only be reflected for future transactions,
+        and will not affect annotations for transactions that already exist.
```

Comment:
I think we'll probably want to document this more thoroughly. Can we put a placeholder link or a TODO just to make sure we don't forget about putting a link to docs here?